src/ode/ode_sir_models.jl

```diff
@@ -0,0 +1,39 @@
+function SIR(du,u,p,t)
+    S = u[1] ; I = u[2] ; R = u[3];
+    β = p[1] ; γ = p[2] ; N = p[3]
```

Comment:
`N = p[3]` is not needed here since the population size is preserved in the model. Rather, add `N = S + I + R` in line 2